components/Common/Card/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { useMemo, type ComponentProps, type FC } from 'react';
+
+import AvatarGroup from '@/components/Common/AvatarGroup';
+import Preview from '@/components/Common/Preview';
+import { shortHumanReadableDate } from '@/util/shortHumanReadableDate';
+
+import styles from './index.module.css';
+
+type Author = {
+  firstName: string;
+  lastName: string;
+  src: ComponentProps<typeof AvatarGroup>['avatars'][number]['src'];
+};
+
+export type CardProps = {
+  title: ComponentProps<typeof Preview>['title'];
+  type: ComponentProps<typeof Preview>['type'];
+  subtitle: string;
+  description: string;
+  author: Author;
+  date: Date;
+};
+
+const Card: FC<CardProps> = ({
+  title,
+  type,
+  subtitle,
```

Comment:
Im not sure but instead of adding similar props, it might be better to just send the type value and use the localized version inside the component

Reply:
Thank you!

This is another doubt I had, but was not really sure from the description of this `Common/Card` component, as it suggests including the `subtitle` prop.

There seems to be a relationship between the `type` and the `subtitle` but they're not exactly the same.

There's only three types: 'announcement' | 'release' | 'vulnerability', and in the designs, the type 'vulnerability' is shown as `Vulnerabilities`.

<img width="442" alt="vulnerabilities-subtitle" src="https://github.com/nodejs/nodejs.org/assets/93147810/9fcb918e-f652-4f78-a33d-895a0a90723c">

@ovflowd knowing this, could you confirm if such translation strings should be 'Vulnerabilities', 'Releases', and 'Announcements'? This way, I could add them in this PR, and remove the `subtitle` property as suggested by @canerakdas

Reply:
Nope, I'm thinking like:

Adding enums into the `/i18n/locales/en.json`

```json
"components.common.card.vulnerability": "Vulnerabilities",
"components.common.card.announcement": "Announcements",
"components.common.card.release": "Releases"
```

And use;

```jsx
<FormattedMessage id={`components.common.card.${type}`} />
```

Reply:
I wanted these to be props and the responsibility of the caller because I am not 100% certain that there won't be subtitles OTHER than the `type` values

Reply:
@canerakdas I had understood you correctly, yes. That's what I meant by adding the translation strings.

Got it, @bmuenzenmeyer, thank you! I will leave the `subtitle` prop as-is then.

Reply:
Based on this informations, it would be better to update as I suggested @danielmarcano 
> edit: i had to go back something like 4 years to this post https://nodejs.org/en/blog/uncategorized/10-lts-to-12-lts to find something that wasn't - and it doesn't even render. I suppose it's fine to start with combining `subtitle` and `type` but yes I agree with @canerakdas then it needs to be internationalized

Reply:
Thanks for pointing this out, @canerakdas. I hadn't read the edited version of @bmuenzenmeyer's comment.

I will remove the `subtitle` prop, in that case.